src/components/Item/VisualizationItem/ItemHeaderButtons.js

```diff
@@ -31,16 +31,25 @@ import {
     hasMapView,
     getAppName,
 } from '../../../modules/itemTypes'
+import { useSystemSettings } from '../../SystemSettingsProvider'
 
 const iconFill = { fill: colors.grey600 }
 
 const ItemHeaderButtons = props => {
-    const [menuIsOpen, setMenuIsOpen] = useState(null)
+    const [menuIsOpen, setMenuIsOpen] = useState(props.isOpen)
 
     const { baseUrl } = useConfig()
 
     const { item, visualization, onSelectActiveType, activeType } = props
 
+    const { settings } = useSystemSettings()
+    const {
```

Comment:
What about doing this renaming at the point of the response, so it will be consistent throughout the app? It would probably be done in SystemSettingsProvider.  (And then update the names throughout the app where systemSettings is used)

Reply:
Sounds good. I've refactored and put the logic for the remappings in api/settings (applied in SystemSettingsProvider). Let me know if this is okay, or if you'd like me to implement some other way